### DISCUSSION TITLE
Why I can't still earn pp?

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#22024] [Q&A] [answered]
Why I can't still earn pp? Is earning pp not available in Android yet?

I'm playing on touch input btw.

![Screenshot_2023-01-04-07-34-35-268_com opera browser](https://user-images.githubusercontent.com/105548541/210459024-95334539-858a-408a-91a5-25977ccd781c.jpg)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/9684
